virttest/qemu_vm.py

```diff
@@ -1731,7 +1731,9 @@ def sort_key(dev):
         if devices.has_option('numa hmat-lb,.*'):
```

Comment:
Prefer to fix the issue by not jumping into this if block when no hmat-lb is defined.

Reply:
@huangyum Cause `hmat-lb` is defined based on `guest_numa_nodes`, what do you think about this:
```
        if devices.has_option('numa hmat-lb,.*'):
            for numa_node in params.objects('guest_numa_nodes'):
                numa_params = params.object_params(numa_node)
                if not numa_params.get('numa_hmat_lb'):
                    continue
                nodeid = numa_params['numa_nodeid']
                ...
```

Reply:
I don't have better idea. This looks good to me.